src/components/run-button-container.js

```diff
@@ -280,8 +207,8 @@ export function RunButtonContainer({ studyUuid, currentNode, disabled }) {
                                 currentNode?.id,
                                 limitReductionParam / 100.0
                             ),
-                        () => setRanLoadflow(true),
-                        () => setRanLoadflow(false),
+                        () => {},
+                        () => {},
```

Comment:
here you can put null
```suggestion
                        null,
```

Reply:
done